tests/test_aws.py

```diff
@@ -195,3 +195,40 @@ def test_metrics(self):
         self.assertTrue(isinstance(sink, aws.MetricsOutput))
         sink.put_metric('ResourceCount', 101, 'Count')
         sink.flush()
+
+
+class OutputLogsTest(BaseTest):
+
+    def test_log_handler(self):
+        session_factory = self.replay_flight_data(
+            'test_log_handler')
+        conf = Bag({
+            'region': 'us-east-2',
+            'scheme': 'aws',
+            'netloc': 'master',
+            'path': 'custodian'
+        })
+        ctx = Bag(session_factory=session_factory,
+            options=Bag(account_id='001100', region='us-east-1'),
+            policy=Bag(name='test', resource_type='ec2'))
+        output = aws.CloudWatchLogOutput(ctx, conf)
+        stream = output.get_handler()
+        self.assertTrue(stream.__dict__['log_group'] == 'custodian')
```

Comment:
for attribute access in assertions,, rather than `stream.__dict__['log_group']` just use stream.log_group